
import two


def main():
    try:           
        # a = two.add(19, "a")
        # if (a):
        #     print("success")
        #     return a
        # else:
        #     print(a)
        print("a"/22)
         
    except Exception as e:
        print(e)
    else:
         print("sinside main functio")
    finally:
        print("done")

main()